api/v1.inc

```diff
@@ -37,6 +39,9 @@ $router->add_route("GET", "v1/projects/:projectid/pages", "api_v1_project_pages"
 $router->add_route("GET", "v1/projects/:projectid/pagedetails", "api_v1_project_pagedetails");
 $router->add_route("GET", "v1/projects/:projectid/pages/:pagename/pagerounds/:pageroundid", "api_v1_project_page_round");
 
+$router->add_route("GET", "v1/queues/:roundid", "api_v1_queues");
+$router->add_route("GET", "v1/queue/:queueid", "api_v1_queue");
```

Comment:
Our convention, for better or worse, has been to use plural nouns even for pulling a singular object (eg: `v1/projects/:projectid`. For consistency, we might consider using:
```
v1/queues/rounds/:roundid
v1/queues/:queueid
```
That mirrors how we do stats too: `v1/stats/site/rounds/:roundid`.